azurerm/resource_arm_managed_disk.go

```diff
@@ -102,6 +102,18 @@ func resourceArmManagedDisk() *schema.Resource {
 			"encryption_settings": encryptionSettingsSchema(),
 
 			"tags": tagsSchema(),
+
+			"disk_iops_read_write": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+
+			"disk_mbps_read_write": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
It would be awesome if you could add these attributes to the Resource documentation too.

Reply:
Thanks for your suggestion. I've added the description to md file. Thanks.